scipy/sparse/_base.py

```diff
@@ -1505,7 +1505,15 @@ def getrow(self, i):
 
 class sparray:
     """A namespace class to separate sparray from spmatrix"""
-    pass
+    
+    
+    def __init__(self, *args, **kwargs):
+        if np.isscalar(args[0]):
+            raise ValueError(
+                "scipy sparse array classes do not \
+                    support instantiation from a scalar"
```

Comment:
The linter complains about missing/extra spaces here. Avoid splitting b/c it fits in the 88 char linting limit.
```suggestion
                "scipy sparse array classes do not support instantiation from a scalar"
```